### ISSUE TITLE
False positive on relative URL paths for "operator-no-unspaced"

### ISSUE THREAD

**Issue by brendanfalkowski** [kristerkari/stylelint-scss#292]
Hi @kristerkari — it's been a few years! I've added these rules into all my SCSS projects and it's a big help.

Just noticed there's a false positive for the `operator-no-unspaced` rule matching some relative file paths. I thinks the `/` is a division operator.

![screen shot 2018-12-20 at 11 36 03 am](https://user-images.githubusercontent.com/214924/50306876-86ac2e00-044b-11e9-9d0e-b0801e6dfa4d.png)

```css
.thing {
    background-image: url(../../img/build/svg/arrow-11-down-dark.svg);
}
```

**Comment by kristerkari** [comment]
Hey @brendanfalkowski! Nice to hear from you.

Thanks for the bug report, I'll have a look.

**Comment by kristerkari** [comment]
PR with a fix:
https://github.com/kristerkari/stylelint-scss/pull/293

**Comment by kristerkari** [comment]
fix included in v3.4.2
https://github.com/kristerkari/stylelint-scss/releases/tag/3.4.2
